packages/grid/backend/grid/core/celery_app.py

```diff
@@ -14,3 +62,7 @@
     "grid.worker.msg_without_reply": "main-queue",
     "delivery_mode": "transient",
 }
+celery_app.worker_prefetch_multiplier = 1
+celery_app.conf.accept_content = ["application/syft"]
```

Comment:
MAYBE:  could you add it in the config file

Reply:
https://github.com/OpenMined/PySyft/blob/dev/packages/grid/backend/grid/core/celery_config.py

Reply:
yeah let me address this in a small PR so i can get this one merged and fix CI